packages/dev/core/src/Materials/greasedLinePluginMaterial.ts

```diff
@@ -0,0 +1,707 @@
+import { Engine } from "../Engines/engine";
+import { RawTexture } from "./Textures/rawTexture";
+import { MaterialPluginBase } from "./materialPluginBase";
+import type { Scene } from "../scene";
+import type { UniformBuffer } from "./uniformBuffer";
+import type { Vector2 } from "../Maths/math.vector";
+import { TmpVectors } from "../Maths/math.vector";
+import { Color3 } from "../Maths/math.color";
+import type { Nullable } from "../types";
+import type { Material } from "./material";
+import { MaterialDefines } from "./materialDefines";
+import type { AbstractMesh } from "../Meshes/abstractMesh";
+import type { BaseTexture } from "./Textures/baseTexture";
+import { DeepCopier } from "../Misc/deepCopier";
+import { RegisterClass } from "../Misc/typeStore";
+
+export enum GreasedLineMeshMaterialType {

```

Comment:
missing typedoc comment (on the values as well)

@RaananW are we not checking those ?

Reply:
Zero warnings in Visual Studio Code. It needs to be commented for sure. I'll add all the missing comments...